src/Slider.jsx

```diff
@@ -270,7 +270,7 @@ class Slider extends React.Component {
 
     const points = Object.keys(marks).map(parseFloat);
     if (step !== null) {
-      const closestStep = Math.round(val / step) * step;
+      const closestStep = (Math.round((val - min) / step) * step) + min;
```

Comment:
This change came directly from the origin repo: [react-component/slider](https://github.com/react-component/slider/commit/e2504d99fe845d686ac468279b5df5008cc8ed04) so it should be good to go. Unless you're asking me to submit a PR to react-component for the upper/lower handle name stuff (which they may or may not accept).